en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+- **Errors by Category** (graph)
+
+    This graph shows the error category distribution over time for the selected period. Apply the required filters as explained above to select the APIs and the error categories to which this content applies. When you select multiple APIs, the error count is grouped by category.
+
+    ![Errors by category graph](../assets/img/insights/error-category-graph.png){.cInlineImage-full}
+    
+- **Errors by Category** (table)
+
+    This table provides further information about the errors such as application details and the error reason. For some authentication errors, the application name is not available. You can use this table to get more concrete information about the errors related to your APIs and then start the problem isolation process.
+ 
+    ![Errors by category table](../assets/img/insights/error-category-table.png){.cInlineImage-full}
+    
+#### Status Code mode
+    
+The following widgets are available for monitoring errors when you have selected the status code mode via the toggle switch in the upper-right corner of the page.
+
+- **Errors by Status Code**
+
+    This graph shows the distribution of HTTP status codes (of the errors) over time for the selected period. Apply the required filters as explained above to select the APIs and status code groups to which this content applies. When you select multiple APIs, the error count is grouped by the status code.
+    
+    ![Errors by status code](../assets/img/insights/error-by-status-code.png){.cInlineImage-full}
+    
+- **Proxy Errors**
+
+    Proxy errors heatmap shows the HTTP response codes of errors that are sent to the client by the API Gateway. Each status code is shown against the selected APIs. The cell color is determined by the number of errors belonging to each status code. The cell color is red for higher numbers and white for lower numbers. Use this chart to further drill down on error code groups that you identified with the **Errors by Status Code** graph.
+    
+    ![Proxy errors](../assets/img/insights/proxy-error-heatmap.png){.cInlineImage-full}
+    
+- **Target Errors**
+
+    Target errors heatmap shows the HTTP response codes of errors that are received by the API Gateway from the backend. Each status code is shown against the selected APIs. The cell color is determined by the number of errors belonging to each status code. The cell color is red for higher numbers and white for lower numbers. Use this chart to further drill down on error code groups that you identified with the **Errors by Status Code** graph.
+
+     ![Target errors](../assets/img/insights/target-error-heatmap.png){.cInlineImage-full}
+    
+### Latency
+
+This page shows information related to the latency of API calls within the API management deployment. You can view a summary of the slowest APIs and then drill down into the API view for further analysis. Use this page as the starting point for debugging API slowness.
```

Comment:
API Management deployment?

Reply:
Suggestion with regard to last sentence:
Use this page as a starting point to debug API slowness.

Reply:
Done